__react/utils/bindings.js

```diff
@@ -0,0 +1,79 @@
+export function includeUnsubAfterForSubscription(message) {
+  if (hasNatsBindings(message) && message.bindings().nats().unsubAfter()) {
+    return `subscribeOptions.max = '${message.bindings().nats().unsubAfter()}';`;
+  }
+  return '';
+}
+export function includeQueueForSubscription(message) {
+  if (hasNatsBindings(message) && message.bindings().nats().queue()) {
+    return `subscribeOptions.queue = '${message.bindings().nats().queue()}';`;
+  }
+  return '';
+}
+
+/**
+ * Does an object have bindings
+ */
+export function hasNatsBindings(obj) {
+  return obj.bindings && obj.bindings.nats;
+}
+  
+/**
+   * is the channel a publish and subscribe type if nothing is specified default to being pubsub type 
+   */
+export function isPubsub(channel) {
+  const tempChannel = channel._json;
+  if (
+    !tempChannel.bindings || 
+      !tempChannel.bindings.nats ||
+      !tempChannel.bindings.nats.is || 
+      tempChannel.bindings.nats.is === 'pubsub') {
```

Comment:
Same as above. Shouldn't it access `bindings().nats()` instead of `bindings.nats`?

Reply:
Should actually use the other function as well 😄

Reply:
We don't have something like `nats()` :) https://github.com/asyncapi/parser-js/blob/master/lib/mixins/bindings.js